src/classes/AFFL_Affiliations_TDTM.cls

```diff
@@ -495,17 +527,16 @@ public class AFFL_Affiliations_TDTM extends TDTM_Runnable {
 
         if (!mapConIdAccIdPrimary.isEmpty()) {
             //uncheck any existing primary affiliations
-            list<npe5__Affiliation__c> afflForUpdate = new list<npe5__Affiliation__c>();
             list<npe5__Affiliation__c> queryAffl = [SELECT Id, npe5__Contact__c, npe5__Organization__c FROM npe5__Affiliation__c WHERE npe5__Primary__c = TRUE AND npe5__Contact__c IN :mapConIdAccIdPrimary.keySet() AND ID NOT IN :mapConIdAfflIdPrimary.values()];
         
             for (npe5__Affiliation__c affl : queryAffl) {
```

Comment:
as I've been reading more about LDV orgs and queries, I suspect that we should redo this query to remove the NOT IN check, which can't use an index, and instead just ignore the primary affiliations in our subsequent for loop.  In a LDV org with a million contacts, and 1 million+ affiliations, I think this query would time out or get an error about not being selective.  If you want to try the change, add it to this pull request.  If you'd prefer to move on, could you just add a backlog item for this change?  thanks.